composer.json

```diff
@@ -63,7 +63,7 @@
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
     "qtism/qtism": "~0",
     "oat-sa/generis" : ">=14.0.0",
-    "oat-sa/tao-core" : ">=47.0.0",
+    "oat-sa/tao-core" : ">=v48.11.0",

```

Comment:
**nitpick:** While this requirement should not be harmful, it is not correct. The exact version that added the required feature is this one [v48.9.0](https://github.com/oat-sa/tao-core/releases/tag/v48.9.0). I think it should be set accordingly for later reference, so that we will able to have the right picture in several months if we need to figure out the changes made recently.

Reply:
Hey @jsconan should we use v# or just a # in composer json or both are acceptable?

Reply:
both are acceptable, but if possible better only use `#`